vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -131,7 +126,7 @@
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\cxxreact\JSBundleType.cpp" />
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\cxxreact\JSExecutor.cpp" />
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\cxxreact\JSIndexedRAMBundle.cpp" />
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\cxxreact\MethodCall.cpp" />
+    <ClCompile Include="$(ReactNativeDir)\ReactCommon\cxxreact\MethodCall.cpp" DisableSpecificWarnings="4244;%(DisableSpecificWarnings)" />
```

Comment:
>DisableSpecificWarnings [](start = 79, length = 23)

I would expect that it must be a child element instead of a property. Are sure that it works?

Reply:
Yep. To my understanding this is the standard way to define item metadata, and it definitely is passed to the compiler.

Reply:
Apparently the ability to specify item metadata as an attribute was added in a relatively new version of MSBuild https://docs.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild?view=vs-2019

Reply:
That's pretty cool.  We should look into doing that with our other warning suppressions, and not using the project level ones we are currently doing.  It would make it more obvious what work has to be done to remove them altogether -- since we obviously would like to fix them all upstream, and ensure that we are not adding cases to our own files.

Reply:
I might just do that now since it hopefully shouldn't be a ton of effort. These should only apply to ReactCommon though instead of RNW.